.github/workflows/ci.yml

```
@@ -272,10 +274,11 @@ jobs:
         with:
           images: gresearchdev/siembol-config-editor-ui
           flavor: |
-            latest=${{ github.ref == 'refs/heads/master' }}
+            latest=${{ github.ref == 'refs/heads/master' && !endsWith(steps.info.outputs.version, '-dev') }}
           # We use raw because our release process is not done on tag push (yet)
           tags: |
-            type=raw,value=${{ steps.info.outputs.version }}
+            type=raw,value=${{ steps.info.outputs.version }}q
```

I believe this is an unintended change. 

```suggestion
            type=raw,value=${{ steps.info.outputs.version }}
```

---

same 🤷‍♂️🤦‍♂️ sorry for this